src/averageAges.js

```diff
@@ -15,6 +15,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const menArrNoCentury = people
+    .filter(person => person.sex === 'm')
+    .map(man => man.died - man.born);
+
+  const menArrWithCentury = people
+    .filter(person => person.sex === 'm'
+    && Math.ceil(person.died / 100) === century)
+    .map(man => man.died - man.born);
+
+  let avrAge;
+
+  century === undefined ? avrAge = menArrNoCentury.reduce((sum, age) =>
+    sum + age) / menArrNoCentury.length : avrAge
+     = menArrWithCentury.reduce((sum, age) =>
+      sum + age) / menArrWithCentury.length;
```

Comment:
you also do the same thing here, don't repeat yourself, create a variable that will store needed array and then `reduce` it